tools/Customization/DevHome.Customization/Views/AddRepositoriesView.xaml.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using DevHome.Common.Extensions;
+using DevHome.Common.Services;
+using DevHome.Customization.Models;
+using DevHome.Customization.ViewModels;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+
+// To learn more about WinUI, the WinUI project structure,
+// and more about our project templates, see: http://aka.ms/winui-project-info.
+namespace DevHome.Customization.Views;
+
+public sealed partial class AddRepositoriesView : UserControl
+{
+    public FileExplorerViewModel ViewModel
+    {
+        get;
+    }
+
+    private readonly IExperimentationService experimentationService = Application.Current.GetService<IExperimentationService>();
+
+    public AddRepositoriesView()
+    {
+        ViewModel = Application.Current.GetService<FileExplorerViewModel>();
+        this.InitializeComponent();
+        if (experimentationService.IsFeatureEnabled("FileExplorerSourceControlIntegration"))
+        {
+            ItemsRepeaterForAllRepoPaths.Visibility = Visibility.Visible;
```

Comment:
The more xaml-y way to do this is instead of setting Visiblity.Collapsed in the markup and changing it here, make the visibility a property of the ViewModel and bind to it. Then you don't need to get the experimentationservice in the View at all.